### ISSUE TITLE
Move skip link after cookie banner in example (DAC usability issue)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2337]
DAC: Move skip link after cookie banner in cookies example, and document

Would also be useful to:
- Add Cookie banner, phase banner etc in the blocks exploded view to users can see what they should be used for
- Add a page example to the skip link component docs to see where it's positioned, how it links to main etc.
- Add cookie banner to base page template?

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/715#discussioncomment-3181057_

Page 53 of EQ RAS DAC report July 2022...

----

Skip to content link (usability)
The cookie section may be missed by blind users due to the positioning of the skip to content link.
WCAG Reference:
Usability & GOV.UK Design System requirement.
Issue ID: DAC_Skip_link_usability
URL: https://eq.surveys.onsdigital.uk/questionnaire/introduction-block/
Page title: Page is not available - ONS Business Surveys
Journey: Task 1 step 1
Screenshot:
<img width="448" alt="Screenshot 2022-07-19 at 15 07 49" src="https://user-images.githubusercontent.com/43346934/179770983-e10d8d0d-d32d-4b95-a36d-387232d61b8c.png">

User focus is taken to the skip to content link before the cookies section. The reason this may be problematic is because some blind users may use this feature unaware that the cookies section is present.
If you’re following the GDS pattern then the skip[ to content link is situated after this section allowing users to encounter it.
Current code ref(s): body > div > div > a

```
 <a class="ons-skip-link" href="#main-content">Skip to main content</a>
```

Screen reader comments: “I explored the start of the page in context and noted that the ‘Skip to main content’ link appeared before the cookie banner. This could result in the cookie banner being bypassed by some screen reader users who decide to jump straight to the page’s main content. In addition, this behaviour does not comply with the GOV.UK Design system best practice. To address this concern, the ‘Skip to main content’ link should preferably appear following the cookie banner. The issue is present with JAWS, NVDA, VoiceOver, TalkBack.”

Solution:
We recommend repositioning the skip to content link so that it is encountered after the cookies section.
<img width="425" alt="Screenshot 2022-07-19 at 15 07 53" src="https://user-images.githubusercontent.com/43346934/179770954-6bfe69a2-bc7d-48bc-b88f-0e35d5fff78c.png">

Example: 
```
<header class="ons-header " role="banner"> <a class="ons-skip-link" href="#main-content">Skip to main content</a>
```

Please refer to the GOV.UK Design System homepage for further guidance.
